src/components/Legend/LegendContainer.tsx

```diff
@@ -22,5 +33,48 @@ export default function LegendContainer({ items, loading = false }: LegendContai
         }))}
       />
     </div>
+  ) : (
+    <>
+      <div className="absolute z-50 bottom-3 right-16">
+        <Tooltip text={INFO_POPUP_TOOLTIP}>
```

Comment:
what is the point of a tooltip if we are on a mobile device?

Reply:
Since in our mechanism we only know it is in mobile version depends on the resolution, when screen width smaller than 640 px e.g. If the use open the website in small screen in laptop, then we need show tooltip although is very unusual.